src/core/src/op/prelu.cpp

```diff
@@ -2,85 +2,79 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "ngraph/op/prelu.hpp"
-
-#include <ngraph/validation_util.hpp>
+#include "openvino/op/prelu.hpp"
 
+#include "element_visitor.hpp"
 #include "itt.hpp"
 #include "openvino/reference/prelu.hpp"
 
-using namespace std;
+namespace ov {
+namespace op {
+namespace prelu {
 
-ov::op::v0::PRelu::PRelu() : Op() {}
+struct Evaluate : element::NoAction<bool> {
+    using element::NoAction<bool>::visit;
 
-ov::op::v0::PRelu::PRelu(const Output<Node>& data, const Output<Node>& slope) : Op({data, slope}) {
-    constructor_validate_and_infer_types();
-}
+    template <element::Type_t ET, class T = fundamental_type_for<ET>>
+    static result_type visit(const Tensor& arg,
+                             const Tensor& slope,
+                             Tensor& out,
+                             const Shape& arg_shape,
+                             const Shape& slope_shape) {
+        reference::prelu(arg.data<const T>(), slope.data<const T>(), out.data<T>(), arg_shape, slope_shape);
+        return true;
+    }
+};
+}  // namespace prelu
+
+namespace v0 {
+
+PRelu::PRelu() : Op() {}
 
-bool ngraph::op::v0::PRelu::visit_attributes(AttributeVisitor& visitor) {
-    OV_OP_SCOPE(v0_PRelu_visit_attributes);
```

Comment:
@praasz could you also take a look how this affects CC in scope of the corresponding task?

Reply:
Yes, I mean conditional compilation, and it isn't a blocker for merge